sdks/python/apache_beam/io/gcp/bigquery_file_loads_test.py

```diff
@@ -1041,7 +1043,7 @@ def test_bqfl_streaming_with_copy_jobs(self):
 
     # Override these parameters to induce copy jobs
     bqfl._DEFAULT_MAX_FILE_SIZE = 100
-    bqfl._MAXIMUM_LOAD_SIZE = 400
+    bqfl._MAXIMUM_LOAD_SIZE = 200
```

Comment:
reason for this change? if linkable, link to public doc as a comment?

Reply:
was running a bunch of these tests after my changes to verify they were passing and found that the size set here was too high and that copy jobs weren't actually being induced. I lowered the value so that the test runs copy jobs as expected.